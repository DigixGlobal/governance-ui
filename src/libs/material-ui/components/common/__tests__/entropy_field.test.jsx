import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import EntropyField from '../entropy_field';

const formData = {
  test: 'Test name',
};

describe('<EntropyField />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <EntropyField
        formChange={jest.fn}
        formData={formData}
        name="test"
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
