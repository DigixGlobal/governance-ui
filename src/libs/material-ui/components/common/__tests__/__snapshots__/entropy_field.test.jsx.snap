// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EntropyField /> renders correctly 1`] = `
<FormField>
  <Input
    action={
      <EntropyGenerator
        bits={256}
        handleSubmit={[Function]}
      />
    }
    fluid={true}
    name="test"
    onChange={[Function]}
    placeholder="randomBytes(32)"
    type="password"
    value="Test name"
  />
</FormField>
`;
